cmd/ferretdb/main.go

```diff
@@ -395,6 +420,20 @@ func run() {
 		stop()
 	}()
 
+	otelShutdown := setupTracer(logger)
+
+	if otelShutdown != nil {
+		defer func() {
+			shutdownCtx, shutdownCancel := ctxutil.WithDelay(ctx.Done(), 3*time.Second) //nolint:mnd // simple shutdown timeout
+
+			if err := otelShutdown(shutdownCtx); err != nil {
+				logger.Sugar().Errorf("Failed to shutdown OpenTelemetry: %s.", err)
```

Comment:
```suggestion
				logger.Error("Failed to shutdown OpenTelemetry", zap.Error(err))
```
Easier to read the log using zap `Field`.